examples_tests/15.LayeredCubeMapShadows/deriv_map_gen.comp

```diff
@@ -4,10 +4,12 @@ layout (local_size_x = 16, local_size_y = 16) in;
 
 layout (binding = 7) uniform sampler2D bumpMapSampler;
 layout (binding = 0, rg8_snorm) uniform image2D derivativeMapImage;
+layout (binding = 1, rg8_snorm) uniform image2D derivativeMapImage_mm[4];
 
 layout (location = 0) uniform float uHeightScaleFactor = 0.001;
 
 shared float smem[324];//18*18
+shared vec2 smem2[256];//16*16
```

Comment:
re-use `smem`, by basically doubling the smem required you'll reduce occupancy greatly